### PR TITLE
Improve firm ("other org") sign-up workflow

### PR DESCRIPTION
**Pull request by cmsetzer**
This is a second attempt at https://github.com/harvard-lil/perma/pull/3593, which I accidentally closed (oops). Included are the following changes to the firm sign-up process:

* Removes some use cases (journals, faculty) from the [Sign Up](https://perma.cc/sign-up) section and retitles the [Law Firms](https://perma.cc/sign-up/firms) section as "Other Orgs."
* Updates the [Sign Up for Free](https://perma.cc/#landing-signup) section on the Perma landing page to reflect the above changes.
* Updates various copy text and links to reflect the above changes.
* Modifies the sign-up form on the Other Orgs (formerly Law Firms) section to include more detailed information, and includes that information in the email it generates.
* Updates tests to match the updated form.
  + Note: I attempted a fresh rewrite of all the "new firm" tests but once I started pulling on that thread, things quickly ballooned out of scope. What's here is a more modest edit of the existing tests.

## To validate

Run Perma locally from this branch and inspect the updated sections:

* [Sign Up for Free](http://perma.test:8000/#landing-signup) section on Perma landing page
* [Other Orgs sign-up form](http://perma.test:8000/sign-up/firms)

You can validate the form-generated email by submitting the form and then checking your terminal output for the rendered email body.

### CODE REVIEW

**Comment by rebeccacremona** [comment]
Whoops, I think the [CSS commit](https://github.com/harvard-lil/perma/pull/3595/commits/cea4bc39fdfbe4daf3d75971c20042094039fa0f) accidentally got the bundle instead of the source.

I'm assuming there was supposed to be a 
```
.col-sm-by4 {
  width: 100%;
  @include respond-desktop {
    width: 100 / 4 * 1%;
    float: left;
  }
}
```
in `style-responsive.css` 🙂 .

File: perma_web/perma/templates/email/admin/firm_request.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+Organization name: {{ organization_form.cleaned_data.name }}
+Organization email: {{ organization_form.cleaned_data.email }}
+Organization website: {{ organization_form.cleaned_data.website }}
+
+Estimated number of individual accounts: {{ usage_form.cleaned_data.estimated_number_of_accounts }}
+Estimated number of Perma Links created each month (per user): {{ usage_form.cleaned_data.estimated_perma_links_per_month }}
+
+User would be an administrator on the account: {{ user_form.data.would_be_org_admin | yesno }}
+
+This user currently {% if target_user %}has{% else %}does not have{% endif %} an account.
```
Comment (rebeccacremona) [inline perma_web/perma/templates/email/admin/firm_request.txt]: Ah! Tiny catch: `target_user` was renamed to `existing_user` in https://github.com/harvard-lil/perma/pull/3595/commits/b78dbbd221785651333b332fa98f42ae473cacd1#diff-2d23437eefdff0673773dfca3188a79c47219c64852a08809a1f0afb9c5b0c07R1911

File: perma_web/perma/views/user_management.py [hunk 2]
```diff
@@ -1901,39 +1905,62 @@ def sign_up_firm(request):
...
+
+        # If user email in form matches an existing user in database, update user record to include
+        # organization name under `LinkUser.requested_account_note` field
+        if existing_user is not None:
+            organization_name = request.POST.get('name', None)
+            existing_user.requested_account_type = 'firm'
+            existing_user.requested_account_note = organization_name
+            existing_user.save()
+            email_firm_request(request, existing_user)
             return HttpResponseRedirect(reverse('firm_request_response'))
```
Comment (rebeccacremona) [inline perma_web/perma/views/user_management.py:1924]: `firm_request_response` renders `registration/firm_request.html`, which looks like this:
image

I bet we want to switch the wording from firm -> organization.

**Review by rebeccacremona — CHANGES_REQUESTED** [review]
This is such an improvement. Huzzah!

Chris I feel like the preparation of this PR bordered on a hazing-level experience; my gratitude and thanks and embarrassed apologies that it was such a beast!

I noticed a couple tiny omissions (missing css; template variable name change; template text tweak), but everything substantive looks great.

I also wanted to throw one thing out there. The checkbox for "Would you be an administrator on this account?" seems a little odd to me.

image
 
I think most of the time, checkboxes are labeled with statements, not with questions.

I think it would be more natural for this to be a set of radio buttons, "yes" and "no", possibly with "no" selected by default (to match the behavior of the checkbox).

What would you think of that? Or do you think that's a question for Clare?

File: perma_web/perma/templates/email/admin/firm_request.txt (hunk 1 above)
Comment (cmsetzer) [reply perma_web/perma/templates/email/admin/firm_request.txt]: Oops. Fixed in [e41888b](https://github.com/harvard-lil/perma/pull/3595/commits/e41888bb186820b396efcb93b00ed1832ec7869d).

File: perma_web/perma/views/user_management.py (hunk 2 above)
Comment (cmsetzer) [reply perma_web/perma/views/user_management.py:1924]: Good catch! Fixed in [c1a6c7b](https://github.com/harvard-lil/perma/pull/3595/commits/c1a6c7beb0a6354cd7adbb58abde79b5ba662046).

**Comment by cmsetzer** [comment]
Thanks for the sanity check and spotting my oversights — all should be fixed now.

I agree about the semantics of the "Would you be an admin?" toggle, and have switched it from a checkbox to a dropdown as we discussed in another thread.

Let me know if this looks good. I'll submit another PR to remove the dead code for journals/faculty.

**Review by rebeccacremona — APPROVED** [review]

**Comment by rebeccacremona** [comment]
🤘

**Pull request merged**